interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -293,6 +304,10 @@ namespace cling {
 
     llvm::LLVMContext* getLLVMContext() { return m_LLVMContext.get(); }
 
+    IncrementalExecutor* getIncrementalExecutor() { return m_Executor.get(); }
+
+    void setExternalIncrementalExecutor(IncrementalExecutor* extIncr);
```

Comment:
Yes, this is because they are used for the set-up of the second 
interpreter.

On 14/12/15 18:03, Axel Naumann wrote:
>
> In interpreter/cling/include/cling/Interpreter/Interpreter.h 
> <https://github.com/karies/root/pull/145#discussion_r47524579>:
>
> > @@ -293,6 +304,10 @@ namespace cling {
> >
> >      llvm::LLVMContext* getLLVMContext() { return m_LLVMContext.get(); }
> >
> > +    IncrementalExecutor* getIncrementalExecutor() { return m_Executor.get(); }
> > +
> > +    void setExternalIncrementalExecutor(IncrementalExecutor* extIncr);
>
> What do we need this getter and setter for? They seem to be unused, 
> and I don't see a need for them.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/karies/root/pull/145/files#r47524579>.
>